{% extends "layout.html" %}
  
{% block title %}Engineering Success{% endblock %}
{% block lead %}Demonstrate engineering success in a part of your project by going through at least one iteration of the engineering design cycle.{% endblock %}

{% block page_content %}



      <p>During the experimental phase of our project, we applied the principles of the engineering cycle to structure our project flow, experimental design, and problem-solving approach. This systematic framework greatly aided us in tackling challenges encountered during the cloning process. In the following section, we will describe how we effectively implemented our project by incorporating the key components of engineering design thinking.</p>
      <h1>1. Design</h1>
      <br>
      <b><h3>Inspirtation of the project</b></h3>

      <p>In recent local news, there has been a growing concern about indoor air quality (IAQ) and the need to study the health effects of indoor air pollution. The Hong Kong Environmental Protection Department has identified various significant air pollutants present in homes, with formaldehyde being one of the most prevalent. Research studies have revealed that formaldehyde can originate from multiple sources, including the combustion of fuels in gas stoves, building and furnishing materials, renovation activities, new wooden furniture, and even the food we consume. This highlights the importance of investigating the impact of formaldehyde on indoor air quality and its potential health implications.</p>
      <P>After consulting with our science teachers, we have gained valuable insights and developed a preliminary idea and direction for designing our project. While we are incorporating synthetic biology concepts into our work, it is crucial to consider that our project's target audience is the general public. Therefore, our main focus should be on creating products that are user-friendly, accessible, and do not require any specialized equipment. We aim to develop solutions that can be easily understood and embraced by the public, emphasizing simplicity and practicality in our design approach. By considering the needs and preferences of the general population, we can ensure that our project resonates with a wider audience and has the potential for real-world impact.</p>
      <p>Consequently, our objective is to develop a device that offers an accessible and user-friendly solution for individuals to monitor formaldehyde levels in their homes. Our aim is to create a device that enables users to easily assess the presence of formaldehyde by simply looking at the device.</p>
      <h3><p><b>Design of constructs</b></p></h3>
      <p><u>Choice of formaldehyde sensing promoters / parts</p></u>
      <br>
      To explore the potential of formaldehyde-inducible promoters and reporters for creating a household formaldehyde sensing device, we turned to the iGEM Parts Registry. We decided to focus on two well-documented parts: the pFrmR promoter (BBa_K2728001) (Fig. 2) designed by iGEM team BGIC-Global in 2018, and the HxlR operon (BBa_K1334002) designed by iGEM team WHU-China in 2014.
      <p>Our objective was to compare the effectiveness of these two parts and determine which one would better serve our purpose. By examining their performance and characteristics, we aimed to identify the most suitable part for our household formaldehyde sensing device.</p>
      <img src="https://static.igem.wiki/teams/4813/wiki/engineering-success/engineering-2.jpeg" height="200" width="400">
      <br>
      <u>Fig. 2 The mechanism of formaldehyde activation of pFrmR promoter. Adopted from Team iGEM18_BGIC-Global Part Regirstry Page</u>
      <br>
      <a href="http://parts.igem.org/wiki/index.php/Part:BBa_K2728001">http://parts.igem.org/wiki/index.php/Part:BBa_K2728001</a>
      <B><h3>The designs of pFrmR constructs</h3></b>
      <p>The two graphic illustrations below shows the designs of pFrmR constructs that we planned to use for formaldehyde sensing.  </p>
      <img src="https://static.igem.wiki/teams/4813/wiki/engineering-success/enginerring-1.jpeg" height="200">
      <br>
      <h3>The design of HxlR constructs</h3>
      <br>
      The other design is based on a HxlR protein which is a formaldehyde-responsive transcription factor from Bacillus subtilis located upstream of the hxlAB operon (Fig 1) (Zhu et al., 2021 & Yurimoto et al., 2005). In the presence of formaldehyde, wild-type HxlR protein binds to two specific DNA binding sites (BRH1&2) and activates the expression of the downstream hxlAB operon in Bacillus subtilis (Zhu et al., 2021 & Yurimoto et al., 2005). 
      <br>
      <img src="https://static.igem.wiki/teams/4813/wiki/engineering-success/engineering-3.jpeg" height="200">
      <p>When formaldehyde is present, HxlR is triggered to bind to BRH1&2, activating the promoter of our dTomato reporter gene. This subsequently promotes the transcription and expression of our reporter gene in the host E.coli, allowing us to detect the presence of formaldehyde simply by observing if there is a red-colour change in our engineered E.coli cells</p>

   <h3><b>K13A mutation in HxlR</b></h3>

    <p>According to the research by Zhu (2021), when the 13th amino acid in HxlR protein, lysine, was replaced by alanine, the mutated K13A version of HxlR showed a more optimal conformation for DNA binding activity and a higher transcription activation than the wild-type HxlR protein.</p>

    <p>Therefore, besides testing the wild-type HxlR, we also incorporate HxlR-K+13A into our construct design to examine whether HxlR-K13A can lead to an enhanced expression of the reporter genes to give E.coli cells a more obvious red-color change upon formaldehyde detection. If yes, HxlR-K13A can be used in our project as an improved design from the wild-type HxlR, by allowing a higher sensitivity in detecting formaldehyde.</p>

    <h3><b>Constructs Designs</b></h3>
    <br>
    <p>There are a total of 4 constructs composing of HxlR to trigger expression of dTomato for formaldehyde detection in our host E.coli.</p>

    <p>In our construct 1, we used the part BBa_K3332042 (which contains the HxlR gene, a strong constitutive reverse promoter named J23100 for HxlR, the two HxlR binding sites, and the wild-type promoter and RBS site for downstream genes) from a previous iGEM team, but we incorporate our desired reporter gene dTomato coding sequence after the wild-type promoter for expression. The expression of wild-type HxlR protein is naturally driven by a reverse promoter in Bacillus subtilis (Yurimoto et al., 2005). The HxlR protein in both of our constructs 1 (Fig 3) and 2 (Fig 4) is backwardly transcribed driven by a reverse promoter just like the wild-type HxlR gene. Both constructs 1 and 2 are being inserted into a commonly used plasmid pUC19 for cloning.</p>
    <br>
    <img src="https://static.igem.wiki/teams/4813/wiki/engineering-success/engineering-success-fig-3.jpg" height="200">
    <br>
    Our construct 2 is similar to construct 1, the only difference is replacing HxlR with HxlR-K13A. We expect to see HxlR-K13A showing enhanced expression of our reporter chromoprotein, dTomato.
    <br>
     <img src="https://static.igem.wiki/teams/4813/wiki/engineering-success/engineering-success-fig-4.jpg" height="200">
    <p>We cloned our constructs 3  (Fig 5) and 4 (Fig 6) into another plasmid pET28A(+)-FGF2. As mentioned above, wild-type HxlR is naturally backwardly transcribed driven by a reverse promoter in Bacillus subtilis and we hardly found any previous research that has explored the possibility of having HxlR forwardly transcribed. Therefore, to explore whether a forwardly transcribed HxlR can still be expressed and retain its activity, we specially designed our constructs 3 and 4 to have HxlR protein expression driven by a forward promoter, the T7 promoter carried in pET28A(+)-FGF2 plasmid. Moreover, both the T7 promoter and the RBS site in pET28A(+)-FGF2 plasmid are modified to increase protein production yield (Shilling et al., 2020). We believe that activation of our reporter chromoprotein dTomato can be further promoted with a higher concentration of HxlR. Hence, besides allowing forward transcription of HxlR protein, we also utilize the modified T7 promoter to enhance expression of HxlR protein. </p>
    <p>The design of our construct 3 is similar to the construct 1 but having the HxlR gene forwardly transcribed under T7 promoter. </p>
    <img src="https://static.igem.wiki/teams/4813/wiki/engineering-success/engineering-success-fig-5.jpg" height="200">
    <br>
    The design of our construct 4 is similar to our construct 3, but the forwardly transcribed wild-type HxlR gene is replaced by HxlR-K13A coding sequence.
    <br>
    <img src="https://static.igem.wiki/teams/4813/wiki/engineering-success/engineering-success-fig-6.jpg" height="200">
    <h3>Choice of reporter genes</h3>
    <p>In order to create a user-friendly device that can be easily used by individuals without specific knowledge or equipment, we recognized the importance of choosing a reporter that would provide a clear and easily observable signal. Among the various options available, we decided that a chromoprotein reporter would be the simplest and most direct choice.</p>
    <p>After conducting a literature review, we found that red chromoproteins were particularly vivid and easy to express. Given our team's prior experience with the red chromoprotein mCherry, we decided to explore the use of dTomato as it was reported to be an even more intense red chromoprotein compared to mCherry. Further research revealed that tdTomato is an enhanced version of dTomato, although this claim specifically referred to its fluorescence properties rather than its chromoprotein color (Shaner, 2004).</p>
    <p>To determine whether tdTomato also exhibited a more intense color than dTomato when expressed in bacterial colonies, we planned to compare the two red proteins as part of our project's objectives. To facilitate this comparison, we obtained the codon-optimized coding sequences of dTomato and tdTomato from NCBI. We then incorporated these sequences, along with a commonly used strong constitutive promoter (BBa_J23100), into two constructs. These constructs were ordered as gBlocks from IDT. Additionally, we also ordered constructs that contained the pFrmR promoter and HxlR operon driving the expression of dTomato and tdTomato, respectively, from IDT. These constructs would enable us to examine the performance of dTomato and tdTomato under the control of different promoters.</p>
    <h3>Choice of biological system</h3>
    <p>We decided to use E. coli as our main organism for this project because it is easily accessible in our laboratory and we are familiar with working with it. An additional advantage is that the pFrmR promoter, which we are using, naturally functions in E. coli since it was discovered in this bacterium. This simplifies our experimental setup as we don't need to introduce an additional protein.</p>
    <p>Now that we have chosen E. coli as our host organism, we can move on to the building phase of our project.</p>
    <h1>2. Build</h1>
    <h3>Cloning of constructs</h3>
    <p>We chose to use the HiFi-assembly method from NEB (New England Biolabs) for our cloning. This method allows us to create the constructs without leaving any scars. Thanks to NEB's sponsorship, we received a free HiFi assembly kit, making the process easier for us. We designed 20-base pair overlapping regions (BBa_K4813007, BBa_K4813008) in our constructs, which successfully assembled into the EcoRI restriction site of the pUC19 vector. This confirmed the effectiveness of the HiFi-assembly method for our cloning purposes.</p>
    <p>We encountered difficulties in cloning our formaldehyde sensing constructs at first. However, by using engineering thinking skills, we were able to solve the problem systematically. We identified the issue, analyzed the process, troubleshooted, made improvements, and validated our modified approach.</p>
    <h2>Cloning of pFrmR constructs (BBa_K4813002, BBa_K481300) and reporter gene comparison (BBa_K4813005, BBa_K4813006) constructs </h2>
    <h3>1st attempt:</h3>
    <p>This was our first time using the HiFi assembly kit and method, and we faced a few challenges during the process. Since we didn't have a spectrophotometer to measure vector concentration, we had to estimate it from gels. </p>
    <p>Initially, we believed our cloning attempt was successful because the HiFi assembly positive controls and the colonies transformed with J23100 constructs (BBa_K4813005 and BBa_K4813006) showed red colors (Fig. 7). Encouraged by this, we proceeded with the double enzyme digestion check of our clones (please refer to the detailed results and experiment page for more information).
</p>
{% endblock %}
